packages/sdk-browser/README.md

```diff
@@ -12,27 +12,126 @@ Turnkey API documentation lives here: https://docs.turnkey.com.
+| [Email Auth](https://docs.turnkey.com/embedded-wallets/sub-organization-auth)         | [auth.turnkey.com](https://auth.turnkey.com)         |
+| [Email Recovery](https://docs.turnkey.com/embedded-wallets/sub-organization-recovery) | [recovery.turnkey.com](https://recovery.turnkey.com) |
+| [Import Wallet](https://docs.turnkey.com/features/import-wallets)                     | [import.turnkey.com](https://import.turnkey.com)     |
+| [Export Wallet](https://docs.turnkey.com/features/export-wallets)                     | [export.turnkey.com](https://export.turnkey.com)     |
+
+#### Wallet
+
+The Wallet client is designed for using your Solana or EVM wallet to stamp and approve activity requests for Turnkey's API.
+This stamping process leverages the wallet's signature to authenticate requests.
+
+The example below showcases how to use an injected Ethereum wallet to stamp requests to Turnkey's API.
+The user will be prompted to sign a message containing the activity request payload to be sent to Turnkey.
+
 ```typescript
 import {
-  TurnkeyBrowserSDK,
-  TurnkeySDKBrowserConfig,
-  TurnkeySDKBrowserClient,
-} from "@turnkey/sdk-browser";
+  createWalletClient,
+  custom,
+  recoverPublicKey,
+  hashMessage,
+} from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { mainnet } from "viem/chains";
+
+import { WalletStamper } from "@turnkey/wallet-stamper";
+
+// Create a new wallet client with a JSON-RPC account from the injected provider
+const walletClient = createWalletClient({
+  chain: mainnet,
+  transport: custom(window.ethereum!),
+});
 
-// This config contains parameters including base URLs, iframe URLs, org ID, and rp ID (relying party ID for WebAuthn)
-import turnkeyConfig from "./turnkey.json";
+const signMessage = async (message: string) => {
+  const account = walletClient.account.address;
+  return walletClient.signMessage({
+    account,
+    message,
+  });
+};
 
-// Use the config to instantiate a Turnkey Client
-const turnkeyClient = new TurnkeyBrowserSDK(turnkeyConfig);
+const getPublicKey = async () => {
+  // Required to recover the secp256k1 public key from the signature
+  const arbitraryMessage = "getPublicKey";
+  const signature = await signMessage(arbitraryMessage);
 
-// Now you can make authenticated requests!
-const response = await turnkeyClient?.passkeySign.login();
+  const secp256k1PublicKey = recoverPublicKey({
+    hash: hashMessage(arbitraryMessage),
+    signature: signature as Hex,
+  });
+  return secp256k1PublicKey;
+};
+
+const walletClient = turnkey.walletClient({
+  wallet: {
+    signMessage,
+    getPublicKey,
+  },
+});
+
+// Make authenticated requests to Turnkey API, such as listing user's wallets
+// User will be prompted to sign a message to authenticate the request
+const walletsResponse = await walletClient.getWallets();
 ```
 
 ## Helpers
```

Comment:
cc @omkarshanbhag probably makes sense to add links to your upcoming docs here :)